### DISCUSSION TITLE
Breadcrumb arrow direction not appearing as expected

### DISCUSSION THREAD

**Discussion by minjidev** [saadeghi/daisyui#1370] [Q&A]
Hello, I'm trying to use breadcrumbs UI, and the arrow direction show upwards.
All the CSS values for related elements seem to be the same, so I am confused why it appears like that.
Any idea why this happens? Thank you! 

![image](https://user-images.githubusercontent.com/68722909/202840552-5bb77e3c-c9c5-4228-808d-f203e175bc88.png)

**Comment by saadeghi** [comment]
It works as expected for me. Tailwind Play example: https://play.tailwindcss.com/EejQWpUMt7

Can you share a code example on Tailwind Play or a small repo so I can reproduce the issue?
